cmds/webboot/types.go

```diff
@@ -54,3 +64,13 @@ type DirOption struct {
 func (d *DirOption) Label() string {
 	return d.label
 }
+
+// BackOption let user back to the upper menu
+// expected to implement Entry interface
+type BackOption struct {
+}
+
+// Label is the string this iso displays in the menu page.
+func (b *BackOption) Label() string {
+	return "Go Back"
+}
```

Comment:
to ensure that things implement the interface you can do this
var _ = Entry(&BackOption{})

Then you don't need to say 'expected to implement' all the time, the code will make that clear.

Reply:
Ok, thank you~